components/email/src/Email.svelte

```diff
@@ -515,12 +515,12 @@
 
   async function handleReplyClick(
     event: MouseEvent,
+    message: Message,
     type: "reply" | "reply_all",
-    msgIndex: number,
   ) {
     event.stopImmediatePropagation();
 
-    const currentMessage = activeThread.messages[msgIndex];
+    const currentMessage = message;
```

Comment:
Do we still need `currentMessage` defined, or can we just use `message` everywhere now?